examples/convert_group_studies.py

```diff
@@ -26,53 +22,42 @@
 
 import mne
 from mne_bids import write_raw_bids, make_bids_basename
-from mne_bids.datasets import fetch_faces_data
 from mne_bids.utils import print_dir_tree
 
 ###############################################################################
 # And fetch the data.
-# .. warning:: This will download 7.9 GB of data for one subject!
-
-subject_ids = [1]
-runs = range(1, 7)
+# .. warning:: This will download 1.6 GB of data!
 
-home = op.expanduser('~')
-data_path = op.join(home, 'mne_data', 'mne_bids_examples')
-repo = 'ds000117'
-fetch_faces_data(data_path, repo, subject_ids)
+data_path = mne.datasets.spm_face.data_path()
 
-output_path = op.join(data_path, 'ds000117-bids')
+# Prepare a path to save the BIDS converted data
+output_path = op.join(op.dirname(data_path), 'MNE-spm-face-bids')
 
 ###############################################################################
 # Define event_ids.
 
 event_id = {
-    'face/famous/first': 5,
-    'face/famous/immediate': 6,
-    'face/famous/long': 7,
-    'face/unfamiliar/first': 13,
-    'face/unfamiliar/immediate': 14,
-    'face/unfamiliar/long': 15,
-    'scrambled/first': 17,
-    'scrambled/immediate': 18,
-    'scrambled/long': 19,
+    'faces': 1,
+    'scrambled': 2,
```

Comment:
ummm ... why did you do this?

Reply:
I reset the ds000117 example, it's like it was before. See https://github.com/mne-tools/mne-bids/pull/254/commits/78a216d2e864af3b8d1b81d44105c901809ed3c1